hbase-procedure/pom.xml

```diff
@@ -84,6 +84,10 @@
     <dependency>
       <groupId>org.apache.hbase</groupId>
       <artifactId>hbase-common</artifactId>
+    </dependency>
+        <dependency>
+      <groupId>org.apache.hbase</groupId>
+      <artifactId>hbase-client</artifactId>
```

Comment:
It is for referencing the Abortable. Anyway, I think this can be  purged by doing some more works in ProcedureStoreFactory, to search for different constructors. As the old WALProcedureStore does not make use of the Abortable.

Reply:
Abortable shoudl be in hbase-common?

Reply:
Maybe but anyway, not a big problem here. Can do in another issue.

Reply:
Just to say spent some time untangling modules. This is a reversion.

Its ok. Can address in a targeted effort at untangling modules (and breaking up hbase-server).